src/components/auth/PrivateRoute.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+function PrivateRoute({ component: Component, ...rest }: Object) {
```

Comment:
Use proper `type Props = { ... };` and `PrivateRoute({ component, ...rest }: Props)` here if it's possible.